lapce-data/src/hover.rs

```diff
@@ -135,12 +150,15 @@ impl HoverData {
         let items = Arc::make_mut(&mut self.items);
         // Extract the items in the format that we want them to be in
         *items = match resp.contents {
-            HoverContents::Scalar(text) => vec![HoverItem::from(text)],
-            HoverContents::Array(entries) => {
-                entries.into_iter().map(HoverItem::from).collect()
+            HoverContents::Scalar(text) => {
+                vec![HoverItem::from_marked_string(text, style)]
```

Comment:
For some reason I keep coming back to this code and disliking the single element vectors. Note to self: this is absolutely fine, move on.

Reply:
I like using `SmallVec` (and would use it in several places here) in some of my other projects for that reason.